Week1/homework/js-exercises/logLengtString.js

```diff
@@ -0,0 +1,4 @@
+'use strict'
+let mySentence = "Programming is so interesting!";
+let x = mySentence.length;
```

Comment:
Nice, I would change `x` to a name that is more meaningfull, for example: `mySentenceLength`

Reply:
yeah I understand esp. when we are working on a big project, meaningfull name would be useful, thank you.